doc/classes/EditorNode3DGizmoPlugin.xml

```diff
@@ -59,6 +59,9 @@
 			</argument>
 			<argument index="1" name="billboard" type="bool" default="false">
 			</argument>
+			<argument index="2" name="icon" type="Texture" default="null">
+				An optional custom icon used to render this handle. If omitted, the default handle texture is used.
```

Comment:
Arguments can't have a description like this, this should be described as part of the method `<description>`.